### PR TITLE
add joltify chain

### PR DESCRIPTION
**Pull request by TheJolter**
Joltify Mainnet: https://app.joltify.io/

### CODE REVIEW

File: cosmos/joltify_1729.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  "chainName": "Joltify",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/joltify_1729/chain.png",
+  "stakeCurrency": {
+    "coinDenom": "JOLT",
+    "coinMinimalDenom": "ujolt",
+    "coinDecimals": 6,
+    "coinGeckoId": "joltify",
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/joltify_1729/ujolt.png"
+  },
+  "walletUrl": "https://app.joltify.io/assets",
```
Comment (editaahn) [inline cosmos/joltify_1729.json:19]: Please remove this line since walletUrl was deprecated.

File: cosmos/joltify_1729.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  "feeCurrencies": [
+    {
+      "coinDenom": "JOLT",
+      "coinMinimalDenom": "ujolt",
+      "coinDecimals": 6,
+      "coinGeckoId": "joltify",
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/joltify_1729/ujolt.png",
+      "gasPriceStep": {
+        "low": 0,
+        "average": 0,
```
Comment (editaahn) [inline cosmos/joltify_1729.json:50]: Just for double-checking, could you verify whether the chain allows a zero gas price?

Comment (TheJolter) [reply cosmos/joltify_1729.json:50]: yes, 0 gas

File: cosmos/joltify_1729.json (hunk 1 above)
Comment (TheJolter) [reply cosmos/joltify_1729.json]: walletUrl removed

**Pull request merged**